projects/project-player-library/src/lib/pages/task-details-page/task-details-page.component.ts

```diff
@@ -1,20 +1,114 @@
 import { Component, OnInit } from '@angular/core';
+import { FormControl, Validators } from '@angular/forms';
+import { MatDialog } from '@angular/material/dialog';
 import { ActivatedRoute } from '@angular/router';
+import { DailogPopupComponent } from '../../shared/dialog-popup/dailog-popup.component';
+import { EditTaskCardComponent } from '../../shared/edit-task-card/edit-task-card.component';
+import { projectDetailsData } from '../../project-details.component.spec.data';
 
 @Component({
   selector: 'lib-taskdetails-page',
-  // standalone: true,
-  // imports: [],
   templateUrl: './task-details-page.component.html',
-  styleUrl: './task-details-page.component.css'
+  styleUrl: './task-details-page.component.css',
 })
-export class TaskdetailspageComponent implements OnInit{
-  constructor(private route:ActivatedRoute){}
+export class TaskDetailsPageComponent implements OnInit {
+  constructor(private route: ActivatedRoute, private dialog: MatDialog) {}
+  taskId: any;
+  selectedValue!: string;
+  textFormControl = new FormControl('');
+  task: any = {};
+  taskOptions = [
+    {
+      viewValue: 'Not Started',
+      value: 'NOTstarted',
```

Comment:
why the value is given as NOTstarted instead of notStarted

Reply:
done